app.py

```diff
@@ -9,13 +9,16 @@
 app.config['SECRET_KEY'] = 'top secret!'
 app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///db.sqlite'
 app.config['OAUTH_CREDENTIALS'] = {
+    # dont forget to register site URL: http://www.qopy.me/Qj5atvdMSMeigE1tNE5aJg
     'facebook': {
-        'id': '470154729788964',
-        'secret': '010cc08bd4f51e34f3f3e684fbdea8a7'
+        'id': '245259309256072',
```

Comment:
Yes, that was done on purpose. You should not make your API credentials public, since that could make anybody impersonate you.